files/bio.md

```diff
@@ -21,6 +21,13 @@
 | Graham Bovett    | gbovett        |
 | Stephen Collins  | SteveCollins17 |
 | Michael Kudla    | michaelwkudla  |
+| Alyssa Kong      | naclsodium     |
+| Anna Cao         | annascao       |
+| Muneeba Ashiq    | muneebaashiq   |
+| Larita Kipkeu    | kipkeu         |
+| Mrunal Mustapure | Moon0256       |
+| Skyler Sattarifard | certainlyskyler|
```

Comment:
Could you add the new names **after** Jake Bobowski, and before Ava?  That way it'll roughly be in chronological order.